testsuite/features/secondary/srv_timezone_feature

```diff
@@ -0,0 +1,140 @@
+    And I enter "galaxy-noise@suse.de" as "email"
+    And I select "(GMT+0800) Malaysia" from "timezone"
+    And I click on "Create Login"
+
+  Scenario: Add roles
+    When I follow the left menu "Users > User List > Active"
+    And I follow "user1"
+    When the "role_satellite_admin" checkbox should be disabled
+    And I check "role_org_admin"
+    And I check "role_system_group_admin"
+    And I check "role_channel_admin"
+    And I check "role_activation_key_admin"
+    And I check "role_config_admin"
+    And I click on "Update"
+
+  Scenario: Login as the new user
+    Given I am authorized as "user1" with password "user1"
+    Then I should see a "user1" link
+
+# bsc 1192699, a P2 bug that is now fixed
+# bsc 1195455, a P3 bug not fixed yet, 
+  Scenario: Schedule a remote script in the future and see the correct timezone as a pop up
+    Given I am on the Systems overview page of this "onalmpantis-min-sles15sp3-1.tf.local"
+    When I follow "Remote Command" in the content area
+    And I enter as remote command this script in
+      """
+      #!/bin/bash
+      ls
+      """
+    And I enter "00:00" as "date_timepicker_widget_input"
+    And I click on "Schedule"
+    Then I should see a "12:00:00 AM MYT" text
+    #WORKAROUND If the above line gets red, they probably fixed the bug, then remove AM from the text
+    
+#  Scenario: Login as the new user if the previous scenario failed
+#    Given I am authorized as "user1" with password "user1"
+#    Then I should see a "user1" link
+
+#WORKAROUND
+# bsc 1195190, a P3 bug not fixed yet, this scenario is disabled as it is a minor bug, if they fix it we will re-enable
+#  Scenario: Cancel the event and see the correct timezone 
+#    Given I am on the Systems overview page of this "onalmpantis-min-sles15sp3-1.tf.local"
+#    When I follow "Events" in the content area
+#    Then I should see a "00:00:00 MYT" text
+#    And I check "list_1697531469_sel"
+#    And I click on "Cancel Selected Events"
+#    Then I should see a "MYT" text  
+
+#  Scenario: Login as the new user if the previous scenario failed
+#    Given I am authorized as "user1" with password "user1"
+#    Then I should see a "user1" link
+
+# bsc 1195191, a P3 bug not fixed yet 
+  Scenario: Schuedule a remote script to run now  and see the correct timezone details in history
+    Given I am on the Systems overview page of this "onalmpantis-min-sles15sp3-1.tf.local"
+    When I follow "Remote Command" in the content area
+    And I enter as remote command this script in
+      """
+      #!/bin/bash
+      ls
+      """
+    And I click on "Schedule"
+    When I follow "Events" in the content area
+    And I follow "History" in the content area
+    When I follow first "Remote Command on onalmpantis-min-sles15sp3-1.tf.local. scheduled by user1"
+    Then I should see a "MYT" text
+    #And I should not see a "PM" text
+    #WORKAROUND remove the comment from the above line if its fixed
```

Comment:
move WORKAROUND comment above the commented out line

Reply:
done